.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Please add /bin to your .gitignore file. It is good practice to only submit source code.

Reply:
Thanks for pointing that out. Will fix that.